src/routes/libraryRoute.ts

```diff
@@ -15,6 +15,9 @@ libraryRouter.get(LIBRARY_ROUTE, getLibraries);
 // Get single library
 libraryRouter.get(`${LIBRARY_ROUTE}/:libraryId`, getSingleLibrary);
 
+// Get game from library
+libraryRouter.get(`${LIBRARY_ROUTE}/:libraryId/:gameId`, getGamefromLibrary);
```

Comment:
Thinking about this more, this might be totally fine but would we prefer to have the route look like this, or should it be a query param?
So for example, if libraryId is 7 and gameId is 2, would we prefer:
1) `libraries/7/2` or
2) `libraries/7?gameId=2

Tagging @ksadoff for another opinion.

Reply:
Good question - did a little reading https://stackoverflow.com/questions/30967822/when-do-i-use-path-params-vs-query-params-in-a-restful-api and looks like query params should be used to filter items, not identify them, so I think better REST practice is to actually do something like `/libraries/{library-id}/games/{game-id}`
This way we're clearly identifying both the library and the game. 

If we were instead fetching games by genre or some other tag then a query param would be appropriate but for unique IDs I think path params would be better

Reply:
yeah i think that makes sense! does that make sense to you too @mshih98 ?